teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, name, parent_permission, specialization)
```

Comment:
- [ ] It will be great to ensure `name` and `parent_permission` have their default values set as `name = "Unknown", parent_permission = true` respectively